src/store/manager/Timeline.js

```diff
@@ -65,7 +65,7 @@ export default class Timeline {
 
   initialFetch() {
     if (!this.initialized) {
-      this.initialized = true
```

Comment:
This line should be left to proper work of keep-alive

Reply:
yeah, but any way to force fetch the latest intead of remain the old data ?